frontend/new-admin-vm/Install-CAM.ps1

```diff
@@ -210,6 +222,48 @@ Configuration InstallCAM
 			MatchSource = $false
 		}
 
+		xRemoteFile Download_Install_Agent.ps1
+		{
+				Uri = "$templateAgentURI/Install-PCoIPAgent.ps1"
+				DestinationPath = "$LocalDLPath\Install-PCoIPAgent.ps1"
+				MatchSource = $false
+		}
+
+		xRemoteFile Download_Install_Agent.sh
+		{
+				Uri = "$templateAgentUri/Install-PCoIPAgent.sh"
+				DestinationPath = "$LocalDLPath\Install-PCoIPAgent.sh"
+				MatchSource = $false
+		}
+
+		xRemoteFile DownloadIdleShutdown.sh
+		{
+				Uri = "$templateAgentUri/$idleShutdownLinux"
+				DestinationPath = "$LocalDLPath\$idleShutdownLinux"
+				MatchSource = $false
+		}
+
+		xRemoteFile sumoagent 
+		{
+				Uri = "$templateAgentUri/$sumoAgentApplicationVM"
+				DestinationPath = "$LocalDLPath\$sumoAgentApplicationVM"
+				MatchSource = $false
+		}
+
+		xRemoteFile sumo 
+		{
+				Uri = "$gitLocation/$sumoConf"
+				DestinationPath = "$LocalDLPath\$sumoConf"
+				MatchSource = $false
+		}
+
+		xRemoteFile Download_Install_Agent.ps1.zip
+		{
+				Uri = "https://teradeploy.blob.core.windows.net/binaries/Install-PCoIPAgent.ps1.zip"
```

Comment:
The blob url is defined as a parameter (sourceUri I think)

Reply:
changed